src/kudu/consensus/consensus_queue.cc

```diff
@@ -999,8 +999,62 @@ void PeerMessageQueue::AdvanceMajorityReplicatedWatermarkFlexiRaft(
 }
 
 std::map<std::string, int> PeerMessageQueue::GetDataCommitQuorum() {
```

Comment:
kuduraft convention is to pass in output variables as ptrs and return Status for function success.